app/javascript/react/src/containers/ShowPage.js

```diff
@@ -0,0 +1,59 @@
+import { Link } from 'react-router';
+import React, {Component} from 'react';
+
+class ShowPage extends Component{
+  constructor(props){
+    super(props)
+    this.state = {
+      celestial: {}
+    }
+  }
+
+  componentDidMount(){
+    let id = this.props.params.id
+    fetch(`http://localhost:3000/api/v1/celestials/${id}`)
+      .then(response => {
+        if (response.ok) {
+          return response;
+        } else {
+          let errorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errorMessage);
+          throw(error);
+        }
+      })
+      .then(response => response.json())
+      .then(response => {
+        this.setState({ celestial: response})
+      })
+      .catch(error => console.error(`Error in fetch: ${error.message}`));
+
+  }
+
+  render() {
+    console.log(this.state)
```

Comment:
Don't forget to remove the console.log on line 33, I'm assuming this is here for testing it locally purposes?